docs/source/api/plugin/landing-pages.md

```diff
@@ -101,6 +101,20 @@ By default, the landing page displays a footer that links to the documentation t
 </td>
 </tr>
 
+<tr>
```

Comment:
Maybe the "This landing page" paragraph should mention you can change it to embed instead of it only being in options? (For both sections)